library/core/src/ptr/non_null.rs

```diff
@@ -1786,21 +1790,18 @@ mod verify {
     use super::*;
     use crate::ptr::null_mut;
 
-    // pub const unsafe fn new_unchecked(ptr: *mut T) -> Self
-    #[kani::proof_for_contract(NonNull::new_unchecked)]
-    pub fn non_null_check_new_unchecked() {
```

Comment:
This should not be deleted. Perhaps the branch wasn't created from the latest commit in main?